rowboat/plugins/infractions.py

```diff
@@ -1053,8 +1327,15 @@ def warn(self, event, user, reason=None):
 
     @Plugin.command('delete', '<infraction:int>', group='infractions', level=CommandLevels.ADMIN)
     def infraction_delete(self, event, infraction):
-        # Written by the wonderful coders over on https://github.com/OGNova/airplane, specifically Tiemen
-        query = "DELETE FROM infractions WHERE id=%s and guild_id = %s and type != 9 RETURNING *;"
+        try:
+            infraction = Infraction.select(Infraction).where(
+                (Infraction.guild_id == event.guild.id) &
+                (Infraction.id == infraction)
+            ).get()
```

Comment:
did you mean to drop the notes type?

Reply:
oops.. Fixed this and also fixed the notes one as the query whole function was a bit broken